tests/performance/apdex-score/specs/test.spec.js

```diff
@@ -0,0 +1,79 @@
+const loadSettings = require('../settings-provider');
+
+const loadPage = require('@page-objects/apdex/load.page');
+const loginPage = require('@page-objects/apdex/login.page');
+const contactsPage = require('@page-objects/apdex/contacts.page');
+const targetsPage = require('@page-objects/apdex/targets.page');
+const tasksPage = require('@page-objects/apdex/tasks.page');
+const messagesPage = require('@page-objects/apdex/messages.page');
+const reportsPage = require('@page-objects/apdex/reports.page');
+
+describe('Apdex Performance Workflows', () => {
+  const settingsProvider = loadSettings();
+  const repetitions = settingsProvider.getIterations();
+  const skip = settingsProvider.getTestsToSkip();
+
+  before(async () => {
+    if (!skip.login) {
+      await loadPage.loadInstance(settingsProvider);
+      await loginPage.login('offline', 'chw', settingsProvider);
+      await loadPage.turnOnAirplaneMode(settingsProvider);
+    }
+  });
+
+  for (let i = 0; i < repetitions; i++) {
```

Comment:
Is the test skipping temporary?

Reply:
yes per test run - however, line 24 just determines how many times we want to run the test suite